app/views/layouts/application.html.haml

```diff
@@ -36,36 +36,58 @@
 
     #page
       - if current_user
-        %nav.nav-left
-          = render 'layouts/nav'
+        - if current_user.roles.present?
+          %nav.nav-left
+            = render 'layouts/nav'
       .container-fluid
         #page-header
           - if current_user
             = link_to(icon('bars') + t('navigation.menu'), '#', class: 'toggle-nav visible-phone')
-            = render 'layouts/quicksearch'
+            - if current_user.roles.present?
+              = render 'layouts/quicksearch'
+            - else
+              = render 'layouts/languages'
             = link_to current_user, person_home_path(current_user), class: 'hidden-phone'
             = link_to t('layouts.nav.sign_out'), sign_out_path, method: "delete", class: 'hidden-phone'
           - elsif current_user.nil? # nil without login, false on error
             = render 'layouts/unauthorized'
 
-        = render_sheets do
-          .content-header
-            %h1
-              = title
-              = help_text_renderer.action_trigger
-            = help_text_renderer.action_text
+        - if current_user && current_user.roles.present?
```

Comment:
Warum brauchts hier zusätzlich einen Check `current_user` und an allen anderen Stellen in dieser Datei nicht? Wäre `current_user?.roles.present?` eine Option?